### PR TITLE
Aggies sprint10 staging

### PR DESCRIPTION
**Pull request by kaladay**
- #120
- #121
- #122
- #124
- #125
- #126

### CODE REVIEW

File: service/src/test/resources/application.yaml [hunk 1]
```diff
@@ -101,9 +101,9 @@ application:
   kafka:
     producer:
       events:
-        topic: ${ENV:folio}.workflow.events
+        topic: ${ENV:folio}.camunda.events
 
-# use to store uploads shared over kafka
+# Used by Kafka to manage uploads.
```
Comment (wwelling) [inline service/src/test/resources/application.yaml:106]: I wasn't entirely happy with the original comment, but this is even more confusing. The `event.upload.path` property is a path to an uploads directory on a mount that is to be shared between mod-workflow and mod-camunda (workflow engine) for when the event to start a workflow has a file upload. The file is stored by mod-workflow and opened by mod-camunda.

File: service/src/main/java/org/folio/rest/workflow/service/OkapiDiscoveryService.java [hunk 2]
```diff
@@ -91,7 +94,7 @@ public Map<String, List<Handler>> getHandlers(String tenant, String id) throws I
   }
 
   public JsonNode getModules(String tenant) {
-    String url = okapiUrl + PROXY_TENANT_BASE_PATH + tenant + MODULES_SUB_PATH;
+    String url = okapiUrl + basePath + PROXY_TENANT_BASE_PATH + tenant + MODULES_SUB_PATH;
```
Comment (wwelling) [inline service/src/main/java/org/folio/rest/workflow/service/OkapiDiscoveryService.java:97]: This is not correct. See line 33 and 34. Okapi requests to `/_/proxy/tenants/` and `/_/proxy/modules/` will be incorrect if prefexid with mod-camunda base-path anything other than `/`. 

Additional but out of scope, `okapiUrl` should have any trailing forward slash removed to be safe.

File: README.md [hunk 3]
```diff
@@ -37,44 +37,46 @@ docker push [docker repo]/folio/mod-workflow:[version]
...
+| DB_PORT                        |           5432              | Postgres port.                                                                                                                                             |
+| DB_QUERYTIMEOUT                |           60000             | Database query timeout.                                                                                                                                    |
+| DB_USERNAME                    |        folio_admin          | Postgres user name.                                                                                                                                        |
+| JAVA_OPTIONS                   | `-XX:MaxRAMPercentage=75.0` | Java options.                                                                                                                                              |
+| KAFKA_HOST                     |           kafka             | Kafka broker host name.                                                                                                                                    |
```
[the remark this answers is not in the document]
Comment (wwelling) [reply README.md:55]: https://english.stackexchange.com/questions/100858/hostname-or-host-name#:~:text=Hostname%20specifically%20refers%20to%20a,a%20specific%20setting%20of%20it.

File: service/src/test/resources/application.yaml [hunk 4]
```diff
@@ -115,14 +115,21 @@ tenant:
...
   initialize-default-tenant: true
+
   domain-packages:
     - org.folio.rest.workflow.components
     - org.folio.rest.workflow.model
   schema-scripts:
 
 okapi:
   url: ${OKAPI_URL:http://localhost:9130}
+  camunda:
```
Comment (wwelling) [inline service/src/test/resources/application.yaml:130]: Could you help explain again why both of these paths are required for mod-workflow to communicate with mod-camunda through Okapi? Is there some limitation with Camunda Dashboard or its configuration for the Camunda API?

**Review by rmathew1011 — APPROVED** [review]

**Pull request merged**